packages/cozy-mespapiers-lib/src/helpers/createPdfAndSave.js

```diff
@@ -57,6 +39,32 @@ export const addCountryValueByQualification = qualification => {
     return acc
   }, {})
 }
+
+/**
+ * @param {object} param0
+ * @param {object} param0.metadata
+ * @param {import('cozy-client/types/types').QualificationAttributes} param0.qualification
+ * @param {string} param0.featureDate
+ * @param {PDFDocument} param0.pdfDoc
+ * @returns {object}
+ */
+export const updateMetadata = ({
+  metadata,
+  qualification,
+  featureDate,
+  pdfDoc
+}) => {
+  return {
+    ...metadata,
+    ...addCountryValueByQualification(qualification),
+    qualification: {
+      ...qualification
+    },
```

Comment:
Is there a reason to use object destructuring on `qualification` ?

Reply:
C'est historique, je ne pense pas que ce soit très utile en effet 👍

Reply:
oui faut justifier le fait qu'on soit développeur (non aucune idée c'est historique et en effet probablement inutile. J'ai d'autre choses à changer je regarderai au passage)